digiwf-spring-cloudstream-utils/src/main/java/io/muenchendigital/digiwf/spring/cloudstream/utils/api/streaming/bpmnerror/service/BpmnErrorService.java

```diff
@@ -0,0 +1,54 @@
+package io.muenchendigital.digiwf.spring.cloudstream.utils.api.streaming.bpmnerror.service;
+
+import io.muenchendigital.digiwf.spring.cloudstream.utils.api.streaming.bpmnerror.dto.BpmnErrorDto;
+import io.muenchendigital.digiwf.spring.cloudstream.utils.api.streaming.infrastructure.StreamingHeaders;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Sinks;
+
+import static io.muenchendigital.digiwf.spring.cloudstream.utils.api.streaming.infrastructure.StreamingHeaders.DIGIWF_PROCESS_INSTANCE_ID;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class BpmnErrorService {
+
+    private static final String MESSAGE_TYPE = "bpmnerror";
+
+    private final Sinks.Many<Message<BpmnErrorDto>> bpmnErrorSink;
+
+    /**
+     * TODO: Sends an error to the bus that can be correlated by process instance ID.
```

Comment:
```suggestion
     * Sends an error to the bus that can be correlated by process instance ID.
```

Is the Todo still required?